Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 ARG BUILDPLATFORM
 
-FROM --platform=$BUILDPLATFORM golang:1.19.6 as builder
+FROM --platform=$BUILDPLATFORM golang:1.20.2 as builder
```

Comment:
It looks like 1.20.3 is being released today with a number of CVE fixes:
https://groups.google.com/g/golang-announce/c/71Wg3N0IZk0/m/xchF2bEZAgAJ

Reply:
Updated.